testrtc.py

```diff
@@ -27,13 +31,49 @@ def get(self):
     content = template.render({})
     self.response.out.write(content)
 
+class UploadHandler(blobstore_handlers.BlobstoreUploadHandler):
+  # Generate and return blobstore upload URL.
+  def head(self):
+    upload_url = blobstore.create_upload_url('/upload')
+    self.response.headers['Response-Text'] = upload_url
+  # Upload file to the blobstore.
+  def post(self):
+    name = self.request.headers['X-File-Name']
+    upload_files = self.get_uploads(field_name=name)
+    blob_info = upload_files[0]
+    host = self.request.headers['Origin']
+    self.response.headers['Response-Text'] = host + '/serve/%s' % blob_info.key()
+
+class ServeHandler(blobstore_handlers.BlobstoreDownloadHandler):
+  def get(self, resource):
+    resource = str(urllib.unquote(resource))
+    blob_info = blobstore.BlobInfo.get(resource)
+    self.send_blob(blob_info)
+
 class TestDownloadFile(webapp2.RequestHandler):
   def get(self, size_kbytes):
     self.response.headers.add_header("Access-Control-Allow-Origin", "*")
     self.response.headers['Content-Type'] = 'application/octet-stream'
     self.response.out.write(random_file[0: int(size_kbytes)*1000])
-    
+
+class CleanBlobstore(blobstore_handlers.BlobstoreUploadHandler):
+  def get(self):
```

Comment:
I've added a check to make sure to only delete blobs with file-names starting with "testrtc-".